README.md

```diff
@@ -256,6 +256,9 @@ Now that you have your CosmosDB all set, you can point the API project to it. In
         "BlobContainerName": "stacks-blob-container-name"
     }
 }
+"AwsSqsConfiguration": {
+    "QueueUrl": "SQS_QUEUE_URL"
+}
 ```
 
 The `SecurityKeySecret` and `ConnectionStringSecret` sections are needed because of our use of the `Amido.Stacks.Configuration` package. `COSMOSDB_KEY`, `SERVICEBUS_CONNECTIONSTRING` or `EVENTHUB_CONNECTIONSTRING` have to be set before you can run the project. If you want to debug the solution with VSCode you usually have a `launch.json` file. In that file there's an `env` section where you can put environment variables for the current session.
```

Comment:
You have to rename that section of the README if you put that there and add the `SQS_QUEUE_URL` explicitly to the text so that it's clear what is what.